src/moderation/notes/notes_minified.go.tmpl

```diff
@@ -0,0 +1,12 @@
.ID) $passwd (toInt $DELETE_TIMEOUT.Seconds)}}{{else}}{{if $passwd := (dbGet .User.ID (printf "notes_delall_%d" $target.ID)).Value}}{{if eq (index .CmdArgs 2) $passwd}}{{dbDel .User.ID (printf "notes_delall_%d" $target.ID)}}{{dbDel $target.ID "notes"}}{{$out.Set "title" (printf "Successfully deleted all notes for %s!" $target)}}{{$out.Set "description" (printf "I've deleted all notes on this user:\n%s (ID %d)" $target $target.ID)}}{{else}}{{$err = "Wrong password. Run the `delall` command again to generate a new one."}}{{dbDel .User.ID (printf "notes_delall_%d" $target.ID)}}{{end}}{{end}}{{end}}{{else if inFold "nuke" $subcommand}}{{if $has_perms = (in (split (index (split (exec "viewperms") "\n") 2) ", ") $NUKE_PERMISSION)}}{{if eq (len .CmdArgs) 1}}{{$passwd := ""}}{{$len := len $PASSWD_CHARSET}}{{range seq 0 12}}{{- $passwd = printf "%s%c" $passwd (index $PASSWD_CHARSET (randInt $len)) -}}{{end}}{{$out.Set "color" 0xffff00}}{{$out.Set "title" "Warning!"}}{{$out.Set "description" "You are about to delete **all** notes on the server. Are you sure you want to do this? There is no going back."}}{{$out.Set "fields" (cslice(sdict "name" "OK. I'm aware and take responsibility." "value" (printf "Good. Run the following command **within the next %s** to confirm your choice.\n```%snote nuke %s```" (humanizeDurationSeconds $DELETE_TIMEOUT) $prefix $passwd))(sdict "name" "I'd like to think about it." "value" "That is also OK. Just let it expire."))}}{{dbSetExpire .User.ID "notes_nuke" $passwd (toInt $DELETE_TIMEOUT.Seconds)}}{{else}}{{if $passwd := (dbGet .User.ID "notes_nuke").Value}}{{if eq (index .CmdArgs 1) $passwd}}{{dbDel .User.ID "notes_nuke"}}{{$out.Set "title" "Started purging entire system."}}{{$out.Set "description" "I will notify you when I'm done."}}{{execCC .CCID nil 10 "exec"}}{{end}}{{else}}{{$err = "Wrong password. Run the `nuke` command again to generate a new one."}}{{end}}{{end}}{{else}}{{$err = printf "You silly, twisted boy, you. (Missing permissions)"}}{{end}}{{else}}{{$err = printf "`%s` is not a valid subcommand!\nValid subcommands are: `%s`.\nRun `%snotes help` for more information." $subcommand $subcommands_string $prefix}}{{end}}{{else}}{{$err = "Member not found."}}{{end}}{{end}}{{else}}{{$err = printf "Just what do you think you're doing Dave? (Missing permissions)" }}{{end}}{{if $err}}{{$err = cembed "description" $err "author" (sdict "name" "An Error Occurred:") "color" 0xff0000"footer" (sdict "text" (printf "Triggered by: %s" .User) "icon_url" (.User.AvatarURL "1024"))}}{{sendMessage nil $err}}{{end}}{{if and $out (not $err)}}{{sendMessage nil (cembed $out)}}{{end}}{{else}}{{$count := dbCount "notes"}}{{if gt $count 0}}{{if ge $count 100}}{{$dump := dbDelMultiple (sdict "pattern" "notes") 100 0}}{{execCC .CCID nil 10 "exec"}}{{else}}{{$dump := dbDelMultiple (sdict "pattern" "notes") $count 0}}{{execCC .CCID nil 10 "exec"}}{{end}}{{else}}{{sendMessage nil (cembed "title" "Successfully purged entire system.")}}{{end}}{{end}}
```

Comment:
CI is failing with
```
unexpected "\"fields\"" in operand
```

Though the action might just be broken, have you checked that this saves and runs properly?

Reply:
Hm, that's interesting. I'll check quickly, all I did was to see if it's below 10000 characters :^)

Reply:
Joe, if you don't mind -- could you run the minifier on that code? It doesn't work for me (perhaps due to my aggressive blocking, I don't want to troubleshoot that right now).

Knowing that you know how to config your minifier, that seems the best shot. There's a plethora of complaints yagpdb is throwing at me and it's not fun to parse through minified code.

The normal large code is fine though.

Reply:
I can take a look tomorrow.

Reply:
Thanks! Appreciate it.

Reply:
Should be good now -- a sneaky `)` sneaked in.